src/main/java/kr/codesquad/controller/ApplicationController.java

```diff
@@ -1,26 +1,66 @@
 package kr.codesquad.controller;
 
+import java.util.Arrays;
+import java.util.List;
 import kr.codesquad.model.Ladder;
-import kr.codesquad.view.InputView;
-import kr.codesquad.view.OutputView;
+import kr.codesquad.view.View;
 
 public class ApplicationController {
-    InputView inputView;
-    OutputView outputView;
+    View view;
     Ladder ladder;
 
-    public ApplicationController(InputView inputView, OutputView outputView, Ladder ladder) {
-        this.inputView = inputView;
-        this.outputView = outputView;
+    public ApplicationController(View inputView, Ladder ladder) {
+        this.view = inputView;
         this.ladder = ladder;
     }
 
     public void run() {
-        outputView.printJoinMembers();
-        String joinMember = inputView.inputUserString();
-        outputView.printMaxLadderHeight();
-        String maxLadderHeight = inputView.inputUserString();
-        String[][] ladderResult = ladder.makeLadder(joinMember, maxLadderHeight);
-        outputView.printLadderResult(ladderResult);
+        String joinMembers = validateName();
+        int maxLadderHeight = Integer.parseInt(validateNumber());
+        List<List<String>> ladderResult = ladder.make(joinMembers, maxLadderHeight);
+        view.printLadderResult(ladderResult);
+    }
+
+    public String validateName() {
+        boolean flag = false;
+        String input = "";
+        while (!flag) {
+            view.printJoinMembers();
+            input = view.inputUserString();
+            flag = checkInputLength(input);
+        }
+        return input;
+    }
+
+    public String validateNumber() {
```

Comment:
```suggestion
    public int validateNumber() {
```
여기도 마찬가지고, 리턴 타입도 int 이면 어떨까요?